src/components/default-blog-dashboard.js

```diff
@@ -0,0 +1,47 @@
+import { graphql } from "gatsby"
+import DefaultLayoutNarrow from "./default-layout-narrow"
+import React from "react"
+import Calendar from "./Calendar"
+export default function BlogDashboard({ data }) {
+  let blogCellsData = data.allMdx.nodes.map(blog=>{
+    return (
+      {
+        slug:blog.fields.slug,
+        name:blog.frontmatter.name,
+        date: blog.frontmatter.date
+      }
+    )
+  })
+
+  console.log(blogCellsData)
+
+  return (
+    <DefaultLayoutNarrow>
```

Comment:
@Mahimahto try replacing this with `<DefaultLayout>` that should take up the full width of the screen.